sample/RedHatSummit17/sessions

```diff
@@ -1,5 +1,5 @@
 [{
```

Comment:
Please compare the object to https://open-event-api.herokuapp.com/#sessions-sessions-details-get & make changes